packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -131,13 +152,24 @@ export function PanelChrome({
 
   const testid = title ? selectors.components.Panels.Panel.title(title) : 'Panel';
 
+  const collapsibleHeader = (
+    <button type="button" className={styles.clearButtonStyles} onClick={toggleOpen}>
+      <Icon name={isOpen ? 'angle-down' : 'angle-right'} />
+      <h6 title={title} className={styles.title}>
+        {title}
+      </h6>
+    </button>
```

Comment:
It is semantically incorrect to place an `h*` inside a `button` element as buttons are categorized as [Prhasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content), which can only contain other phrasing content, while `h*` are Heading content. 

maybe we should do the opposite:

```tsx
<h6><button></button></h6>
```